docs/cases/cases-overview.asciidoc

```diff
@@ -52,13 +53,16 @@ image::images/cases-ui-open.png[]
 [[cases-ui-manage]]
 == Manage existing cases
 
-You can search existing cases, and filter them by tags, reporter, and status
+You can search existing cases and filter them by tags, reporter, and status
 (open, in-progress, or closed).
 
+NOTE: To update cases, your role requires certain Kibana space privileges.  Specifically, your role needs *All* sub-feature privileges for the *Case* feature and *All* privileges for the *Action and Connectors* feature. See <<case-permissions>> for more information.
```

Comment:
The Action and Connectors feature is needed only if you want to push cases to external services. You can update a case without it. The user sees the following screenshot with no connectors permission:

<img width="388" alt="Screenshot 2021-07-19 at 5 22 56 PM" src="https://user-images.githubusercontent.com/7871006/126175718-98c1d72d-4d53-41b6-b8f0-20cd901a2033.png">